### DISCUSSION TITLE
Compile latest version for Windows with ROCm enabled

### DISCUSSION THREAD

**Discussion by screwballl** [lllyasviel/Fooocus#3278] [Q&A]
Looking for assistance compiling latest version for Windows with AMD ROCm enabled. Working well with current setup but would like the latest version to have the fixes. 5600X, 32GB RAM, 8GB RX7600 using AMD PRO 24.10.16 drivers and ROCm SDK installed.
The --directml is so slow compared to CUDA native, aka 2 minutes per image on 8GB RX7600 versus 20 seconds per image with 8GB RTX3050.

**Comment by Rogala** [comment]
[Does ROCm work with Windows?](https://pytorch.org/get-started/locally/)

**Comment by screwballl** [comment]
Yes it does: https://www.amd.com/en/developer/resources/rocm-hub/hip-sdk.html
